docs/migrate-module.md

```diff
@@ -0,0 +1,94 @@
+# Update an older module
+
+Module repos created before July 2022 don't include the logic that is needed to pass the continuous integration (CI) tests and to publish the modules to the IBM Cloud catalog. Follow these steps to update an existing repo in the terraform-ibm-modules GitHub organization.
+
+## Before you begin
+
+- If you're running Microsoft Windows, set up Windows Subsystem for Linux [WSL](https://ubuntu.com/wsl) and run commands within WSL.
+- Make sure that you have a current version of Python 3 and pip.
+- Make sure that you have a recent stable version of [Go](https://go.dev/doc/install) installed and available in your PATH environment variable.
+
+## Create an empty module
+
+Create an empty module repo in the `terraform-ibm-modules` GitHub organization.
+
+1.  In the upper right of any page in the [terraform-ibm-modules](https://github.com/terraform-ibm-modules) organization, use the **+** menu, and select **New repository**.
+1.  Select `terraform-ibm-modules` as the owner.
+1.  Enter a name for the module in format `terraform-ibm-<NAME>`, where `<NAME>` reflects the type of infrastructure that the module manages. Use hyphens as delimiters for names with multiple words (for example, terraform-ibm-`activity-tracker`).
```

Comment:
We could link to the existing module guidelines on naming

Reply:
I'd rather keep this in here because they're already jumping out to the UI while they're reading this.

Reply:
Ok - we just need to ensure we have a process to maintain both locations going forwards

Reply:
process=me

Reply:
I found a way to reuse here and in the contributing topic.